ios/RCTMapboxGL/RCTMapboxGLManager.m

```diff
@@ -153,7 +153,7 @@ - (id)init
     _loadedPacks = NO;
     
     // Setup pack array loading notifications
-    [[MGLOfflineStorage sharedOfflineStorage] addObserver:self forKeyPath:@"packs" options:0 context:NULL];
+    [[MGLOfflineStorage sharedOfflineStorage] addObserver:self forKeyPath:@"packs" options:NSKeyValueObservingOptionInitial context:NULL];
```

Comment:
Does this set up a race condition where the pack notification handlers (set up below) aren't created by the time the initial change notification comes in? It would depend on whether the initial change notification is posted synchronously or whether it is posted asynchronously at the end of the current run loop iteration.